ztp/ztp-ran-policy-generator/sourcePolicies/ClusterLogForwarder.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs: $outputs
+  inputs:
+  - name: infra-logs
+    infrastructure:
+      namespaces:
+      - openshift-apiserver
+      - openshift-cluster-version
+      - openshift-etcd
+      - openshift-kube-scheduler
+      - openshift-monitoring
```

Comment:
I know this is just an example, but we should probably include some of our day-2 namespaces which would be of interest to the customer. I'd suggest:
openshift-performance-addon
openshift-ptp
and perhaps:
openshift-machine-config-operator